javascript-algorithms-and-data-structures/pokemon-search-app/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Pokémon Search App</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <h1>Pokémon Search App</h1>
+    <div class="container">
+      <form role="search" id="search-form">
+        <label for="search-input" class="sr-only"
+          >Search for Pokémon Name or ID</label
+        >
+        <input
+          type="text"
```

Comment:
I think we should add the required attribute because right now this alert shows up if the user does not input anything
<img width="443" alt="Screenshot 2023-08-30 at 9 38 09 AM" src="https://github.com/freeCodeCamp/CurriculumExpansion/assets/67210629/855749ba-474e-48fb-b707-167ba74d260f">

Reply:
I have updated this in the latest commit.